src/main/java/com/minecraftabnormals/upgrade_aquatic/core/mixin/DrownedModelMixin.java

```diff
@@ -15,46 +15,46 @@
 @Mixin(DrownedModel.class)
 public class DrownedModelMixin<T extends ZombieEntity> extends ZombieModel<T> {
 
-	protected DrownedModelMixin(float p_i48914_1_, float p_i48914_2_, int p_i48914_3_, int p_i48914_4_) {
-		super(p_i48914_1_, p_i48914_2_, p_i48914_3_, p_i48914_4_);
+	private DrownedModelMixin(float p_i48914_1_, float p_i48914_2_, int p_i48914_3_, int p_i48914_4_) {
+	    super(p_i48914_1_, p_i48914_2_, p_i48914_3_, p_i48914_4_);
 	}
 
 	@Inject(at = @At("TAIL"), method = "setRotationAngles(Lnet/minecraft/entity/monster/ZombieEntity;FFFFF)V")
 	public void setRotationAngles(T entityIn, float limbSwing, float limbSwingAmount, float ageInTicks, float netHeadYaw, float headPitch, CallbackInfo info) {
```

Comment:
The formatting has improved and the variables inside have been renamed, but there are still some formatting errors inside the method.